publication-commons/src/main/java/no/unit/nva/publication/service/impl/ApprovalAssignmentServiceForImportCandidate.java

```diff
@@ -0,0 +1,118 @@
+package no.unit.nva.publication.service.impl;
+
+import static nva.commons.core.attempt.Try.attempt;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Stream;
+import no.unit.nva.clients.CustomerDto;
+import no.unit.nva.clients.IdentityServiceClient;
+import no.unit.nva.model.Contributor;
+import no.unit.nva.model.Organization;
+import no.unit.nva.publication.model.business.importcandidate.ImportCandidate;
+import nva.commons.core.paths.UriWrapper;
+
+public class ApprovalAssignmentServiceForImportCandidate {
```

Comment:
Is this "Approval" for the the import candidate as a whole, or just the files? If the latter , then maybe `ImportCandidateFileApprovalAssignmentService`

Reply:
Just the files